config/initializers/carrierwave.rb

```diff
@@ -33,7 +33,11 @@ def strip
 
 # Additional carrierwave configurations
 CarrierWave.configure do |config|
-  config.cache_dir = "#{Rails.root}/tmp/uploads"
+  config.cache_dir = Rails.root.join("tmp", "uploads")
+
+  if Rails.env.test?
+    config.enable_processing = false
```

Comment:
Oooh! Does this make our tests faster in general? Nice addition.

Reply:
Yup! Not a huge difference in our case, but it's the approach suggested by the carrierwave docs.